models/model1_part1.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    with zipfile.ZipFile("Datasets.zip","r") as zip_ref:
</pre><p>Any reason this is extracted before the imports?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>There is no specific reason for that, it was just the error encountered while unzipping the zip file in the MarkovML notebook. The first basic !unzip command was not running, we had to use the import zipfile (the second way) for unzipping the file. May I now remove it?</p>